client/src/components/J40Header.tsx

```diff
@@ -89,67 +91,63 @@ const J40Header = () => {
-    // possibilities so it's simple. Note: strings are used as react keys
-    const menu =
-      ('sprint3' in flags!) ?
-        ['about', 'cejst', 'methodology', 'contact'] :
-        ['about', 'cejst', 'methodology', 'contact'];
-    // TODO: make feature flags flags work.
+    const menu =['about', 'cejst', 'methodology', 'contact'];
     return menu.map((key) => menuData.get(key));
   };
 
   return (
     <>
-      <Helmet htmlAttributes={{lang: intl.locale}}>
-        <meta charSet="utf-8"/>
-        <title>{title}</title>
-      </Helmet>
       <GovBanner/>
       <Header
         basic={true} role={'banner'}
         className={'usa-header j40-header'}>
         <div className="usa-nav-container">
           <div className="usa-navbar">
-            <Title>{title}
-              <div className={'byline'}>
-                A climate and economic justice screening tool
-              </div>
-            </Title>
-
+            <h1 className="usa-logo">
+              <img className="sitelogo" src={siteLogo} alt={title} />
+              <span className={'usa-logo__text'}>{title}</span>
+            </h1>
             <NavMenuButton
               key={'mobileMenuButton'}
               onClick={toggleMobileNav}
               label="Menu"/>
           </div>
           <PrimaryNav
-            items={headerLinks(flags)}
+            items={headerLinks()}
             mobileExpanded={mobileNavOpen}
             onToggleMobileNav={toggleMobileNav}
             className={'j40-header'}
           >
           </PrimaryNav>
         </div>
       </Header>
-      <Alert
-        className={'j40-sitealert'}
-        type="info">
-        <b>Public beta — </b>
-        Welcome to the public beta of the Just Progress Map, a climate and
-        economic justice screening tool. The tool will be continuously updated.
-        Please submit feedback.
+
+      <Alert className={'j40-sitealert'} type="info">
+        <b><FormattedMessage
+          id='header.alertTitleBeta'
+          description={'Alerts that appear on every page - title'}
+          defaultMessage={`Public beta`}/> - </b>
+        <FormattedMessage
+          id='header.alertBodyBeta'
+          description={'Alerts that appear on every page'}
+          defaultMessage={`This website will be continuously updated`}/>
         <br/>
       </Alert>
-      <Alert
-        className={'j40-sitealert'}
-        type="warning">
-        <b>Limited data sources — </b>
-        This tool currently includes 16 datasets. Over time, datasets could be
-        added, updated, or removed. The datasets come from a variety of sources
-        based on availability, quality, and relevance to environmental, energy,
-        and climate issues. Each dataset has limitations, such as how recently
-        the data was updated.
-      </Alert>
+
+      { // include this alert ONLY on the maps page.
+        location?.pathname.includes('cejst') ?
```

Comment:
hm, this works, but is maybe a bit icky - it overall feels like the header shouldn't have to know where it is located , for the purposes of abstraction. Also, if we change the pagename, we'd have to know to change it here also. 

I'm wondering whether you investigated including it directly in the `cejst` page itself? this would feel a bit cleaner

Reply:
Yeah, good point. I'll see if I can't move it into the maps content.